gifgen

```diff
@@ -2,32 +2,35 @@
 
 # Echo help/usage message
 show_help() {
-  echo "gifgen 1.1.2"
+  echo "gifgen 1.2.0"
   echo
   echo "Usage: gifgen [options] [input]"
   echo
   echo "Options:"
   echo "  -o   Output file [input.gif]"
   echo "  -f   Frames per second [10]"
+  echo "  -w   Width, horizontal resolution [640]"
   echo "  -s   Optimize for static background"
   echo "  -v   Display verbose output from ffmpeg"
   echo
   echo "Examples:"
   echo "  $ gifgen video.mp4"
   echo "  $ gifgen -o demo.gif SCM_1457.mp4"
   echo "  $ gifgen -sf 15 screencap.mov"
+  echo "  $ gifgen -w 700 -s -o my.gif frames/frame_%4d.png"
 }
 
 # Setup defaults
 pid=$$
 palette="/tmp/gif-palette-$pid.png"
 fps="10"
+width="640"
```

Comment:
We shouldn't default to a custom width. It should use the input width unless specified otherwise.

Reply:
That's how it currently behaves. Like the 10 fps, the 640 is used when no width is passed in based on my tests.